### PR TITLE
rebrand

### PR DESCRIPTION
**Pull request by dailosu**

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -5,27 +5,27 @@
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 
 
-- [Why did you create PeerTube?](#why-did-you-create-peertube)
-- [I don't like the name "PeerTube"](#i-dont-like-the-name-peertube)
+- [Why did you create BitTu?](#why-did-you-create-BitTube)
```
Comment (cabaleiro8) [inline FAQ.md]: BitTu -> BitTube
```suggestion
- [Why did you create BitTube?](#why-did-you-create-BitTube)
```

File: FAQ.md [hunk 2]
```diff
@@ -125,13 +130,12 @@ To check if your BitTorrent client supports WebTorrent you can see this issue: h
 
 
 ## Why host on GitHub and Framagit?
```
Comment (cabaleiro8) [inline FAQ.md]: Remove Framagit
```suggestion
## Why host on GitHub?
```

File: FAQ.md [hunk 3]
```diff
@@ -125,13 +130,12 @@ To check if your BitTorrent client supports WebTorrent you can see this issue: h
...
-
-The project has initially been hosted on GitHub by Chocobozzz. A full migration to [Framagit](https://framagit.org/chocobozzz/PeerTube) would be ideal now that Framasoft supports PeerTube, but it would take a lot of time and is an ongoing effort.
+The project has initially been hosted on GitHub by ipbc-dev.
```
Comment (cabaleiro8) [inline FAQ.md]:
```suggestion
The project has initially been hosted on GitHub by ipbc-dev. We will keep it here as it is more accesible to everybody.
```

**Review by cabaleiro8 — CHANGES_REQUESTED** [review]
Accept changes and I will merge. Thanks!

**Review by cabaleiro8 — APPROVED** [review]

**Comment by ghost** [comment]
Hi dear Bittube devs!

I have a few things to tell you:
-I saw that you're building mobile apps, have you considered forking and rebranding Thorium? It's a great android client for Peertube/Bittube.
-Do you plan to transition bittube.tv to your peertube-based Bittube? When do you plan to do this? Thanks
-Have you considered using GitLab or [Framagit](https://framagit.org)? Github isn't open source.

**Pull request merged**